test/conformance/walk.go

```diff
@@ -85,12 +87,17 @@ func (v *visitor) convertToConformanceData(at *ast.BasicLit) {
 	cd.Description = ""
 	for _, line := range lines {
 		line = strings.TrimSpace(line)
-		if strings.HasPrefix(line, "Testname:") {
+		if ok, _ := regexp.MatchString("^\\s*Testname\\s*:", line); ok {
 			line = strings.TrimSpace(line[9:])
 			cd.TestName = line
 			continue
 		}
-		if strings.HasPrefix(line, "Description:") {
+		if ok, _ := regexp.MatchString("^\\s*Release\\s*:", line); ok {
+			line = strings.TrimSpace(line[9:])
```

Comment:
I guess here should be
```
line = strings.TrimSpace(line[8:])
```
for "Release:" which is 1 char shorter than "Testname:".

Reply:
Fixed. thanks @oomichi